composer.json

```diff
@@ -12,18 +12,19 @@
     "require": {
         "php": "^5.3.9|^7.0",
         "webmozart/path-util": "^2.2",
-        "webmozart/glob": "^4.0",
-        "webmozart/assert": "^1.0"
+        "webmozart/glob": "^4.1",
+        "webmozart/assert": "^1.0",
+        "psr/log": "^1.0"
     },
     "require-dev": {
-        "webmozart/key-value-store": "^1.0.0-beta5",
+        "webmozart/json": "^1.2@dev",
```

Comment:
What do you think of releasing a beta of webmozart/json and use it here? I think it would be great for beta10 to be as stable as possible.